### PR TITLE
Add post on small development grant

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

File: _posts/2022-04-19-small-development-grant.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+The person being funded is [Dr. Benoît Legat](https://github.com/blegat), a core
+contributor to JuMP, and the project builds upon his work developing
+[ComplexOptInterface.jl](https://github.com/jump-dev/ComplexOptInterface.jl).
+
+The grant has three key deliverables:
+
+ * Improve ComplexOptInterface.jl from a proof-of-concept into a robust,
+   production-ready library
+ * Integrate ComplexOptInterface into MathOptInterface and document
```
Comment (blegat) [inline _posts/2022-04-19-small-development-grant.md:35]:
```suggestion
 * Integrate ComplexOptInterface into MathOptInterface, test and document
```

File: _posts/2022-04-19-small-development-grant.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * Integrate ComplexOptInterface into JuMP and document.
```
Comment (blegat) [inline _posts/2022-04-19-small-development-grant.md:36]:
```suggestion
 * Integrate ComplexOptInterface into JuMP, test and document.
```

File: _posts/2022-04-19-small-development-grant.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+The [JuMP Steering Committee](/pages/governance/#steering-committee) is pleased
+to announce that we have received a
+[small development grant](https://numfocus.org/programs/small-development-grants)
+from [NumFOCUS](https://numfocus.org) to add complex number support to JuMP.
+
+Currently, JuMP and MathOptInterface are limited to formulating optimization
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. The most important of these is
```
Comment (joaquimg) [inline _posts/2022-04-19-small-development-grant.md:15]: Maybe not under-rate other aplications of complex numbers...
"One of the most important..."
"One of the most famous..."

File: _posts/2022-04-19-small-development-grant.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+the AC optimal power flow problem (AC-OPF), which is used world-wide to optimize
+the operations of electrical power grids. Currently, users in JuMP are forced to
+decompose their problem into the real and complex parts. This leads to a
+duplication in which twice as many variables are needed, reducing readability
```
Comment (joaquimg) [inline _posts/2022-04-19-small-development-grant.md:19]: wont the variables still be duplicated for practical solvers?

**Review by joaquimg — COMMENTED** [review]
Cool!

File: _posts/2022-04-19-small-development-grant.md (hunk 3 above)
Comment (odow) [inline _posts/2022-04-19-small-development-grant.md:15]:
```suggestion
a variety of industrial optimization problems. One of the most important of these is
```

File: _posts/2022-04-19-small-development-grant.md (hunk 4 above)
Comment (odow) [reply _posts/2022-04-19-small-development-grant.md:19]: Yes, but the problem is mainly in the readability. If it spurs solvers to support complex values natively, that's also a good outcome.

**Review by blegat — APPROVED** [review]

File: _posts/2022-04-19-small-development-grant.md (hunk 4 above)
Comment (blegat) [reply _posts/2022-04-19-small-development-grant.md:19]: Yes, most solvers do not support complex numbers yet

File: _posts/2022-04-19-small-development-grant.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Currently, JuMP and MathOptInterface are limited to formulating optimization
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. One of the most important of these is
+the AC optimal power flow problem (AC-OPF), which is used world-wide to optimize
+the operations of electrical power grids. Currently, users in JuMP are forced to
+decompose their problem into the real and complex parts. This leads to a
+duplication in which twice as many variables are needed, reducing readability
+and increasing the likelihood of bugs.
+
+The lack of complex number support has been a common source of complaint for
```
Comment (mlubin) [inline _posts/2022-04-19-small-development-grant.md:22]: s/common source of complaint/common feature request/

File: _posts/2022-04-19-small-development-grant.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. One of the most important of these is
+the AC optimal power flow problem (AC-OPF), which is used world-wide to optimize
+the operations of electrical power grids. Currently, users in JuMP are forced to
+decompose their problem into the real and complex parts. This leads to a
+duplication in which twice as many variables are needed, reducing readability
+and increasing the likelihood of bugs.
+
+Complex number support has been a common feature request for users of JuMP, and
+many users leave JuMP for alternative tools which have complex-number support. In
```
Comment (mlubin) [inline _posts/2022-04-19-small-development-grant.md:23]: "many users leave JuMP"
What is meant by "many"? 10s? 100s? 1000s?

It doesn't seem necessary to put emphasis on a negative here. It's implied that users requesting this feature might decide to use other tools that provide it.

**Review by mlubin — APPROVED** [review]

**Pull request merged**